environment.yml

```diff
@@ -0,0 +1,20 @@
+name: datapipes
+channels:
+  - pytorch
+  - conda-forge
+  - defaults
+dependencies:
+  - pip
+  - pytorch
+  - rioxarray
+  - torchdata
+  - torchvision
+  - xarray
+  - fsspec
+  - zarr
+pip:
+  - einops
+  - pathy
+  - git+https://github.com/SheffieldSolar/PV_Live-API
+  - pyaml_env
+  - nowcasting_datamodel
```

Comment:
can we get away with just requirements?

Reply:
I like having the environment.yaml for development and an easier install way for rioxarray and other complicated install things like the PV Live api.